www/src/theme/Playground/Preview.tsx

```diff
@@ -55,9 +55,17 @@ const Preview: React.FC<PreviewProps> = ({ className }) => {
     },
   );
 
+
+  const handleCustomRedirect = (e) => {
```

Comment:
Can you wrap this callback with a `useEventCallback` from `@restart/hooks`.  Also add a type to `e` and remove the cast on line 60